CONTRIBUTING.md

```diff
@@ -119,10 +119,10 @@ to your fork until you have satisfied all feedback.
 
 Note: Code review is a burdensome but important part of the development process, and as such, certain types of pull requests are rejected. In general, if the **improvements** do not warrant the **review effort** required, the PR has a high chance of being rejected. It is up to the PR author to convince the reviewers that the changes warrant the review effort, and if reviewers are "Concept NAK'ing" the PR, the author may need to present arguments and/or do research backing their suggested changes.
```

Comment:
while fixing up the typo (repeatetly), perhaps also:

s/"Concept NAK'ing"/"Concept NACK" on/